docs/features/trajectories/trajectories.ipynb

```diff
@@ -0,0 +1,260 @@
+    "It does so by generating a constraint of the following form:\n",
+    "\n",
+    "\\begin{align}\n",
+    "    c = \\mathrm{sign}_a \\mathrm{var}_a + \\mathrm{sign}_b \\mathrm{var}_b\n",
+    "\\end{align}\n",
+    "\n",
+    "Method `add_linkage_constraint` lets the user specify the variables and phases to be compared for this constraint, as well as the location of the variable in each phase (either 'initial' or 'final')\n",
+    "By default this method is setup to provide continuity in a variable between two phases:\n",
+    "- the sign of variable `a` is +1 while the sign of variable `b` is -1.\n",
+    "- the location of variable `a` is 'final' while the location of variable `b` is 'initial'.\n",
+    "- the default value of the constrained quantity is 0.0.\n",
+    "\n",
+    "In this way, the default behavior constrains the final value of some variable in phase `a` to be the same as the initial value of some variable in phase `b`.\n",
+    "Other values for these options can provide other functionality.\n",
+    "For instance, to simulate a mass jettison, we could require that the initial value of `mass` in phase `b` be 1000 kg less than the value of mass at the end of phase `a`.\n",
+    "Providing arguments `equals = 1000, units='kg` would achieve this.\n",
+    "\n",
+    "Similarly, specifying other values for the locations of the variables in each phase can be used to ensure that two phases start or end at the same condition - such as the case in a branching trajectory or a rendezvous.\n",
+    "\n",
+    "While `add_linkage_constraint` gives the user a powerful capability, providing simple state and time continuity across multiple phases would be a very verbose undertaking using this method.\n",
+    "The `link_phases` method is intended to simplify this process.\n",
+    "In the finite-burn orbit raising example, there are three phases:  `burn1`, `coast`, `burn2`.\n",
+    "This case is somewhat unusual in that the thrust acceleration is modeled as a state variable.  \n",
+    "The acceleration needs to be zero in the coast phase, but continuous between `burn1` and `burn2`, assuming no mass was jettisoned during the coast and that the thrust magnitude doesn't change.\n",
+    "\n",
+    "### add_linkage_constraint\n",
+    "\n",
+    "```{eval-rst}\n",
+    "    .. automethod:: dymos.Trajectory.add_linkage_constraint\n",
+    "        :noindex:\n",
+    "```\n",
+    "\n",
+    "### link_phases\n",
+    "\n",
+    "```{eval-rst}\n",
+    "    .. automethod:: dymos.Trajectory.link_phases\n",
+    "        :noindex:\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Examples of using the `link_phases` method\n",
+    "\n",
+    "**Typical Phase Linkage Sequence**\n",
+    "\n",
+    "A typical phase linkage sequence, where all phases use the same ODE (and therefore have\n",
```

Comment:
The 4 examples under "Typical Phase Linkage Sequence" also appear right before this header. I think they are being pulled in from the docstring for `dymos.Trajectory.link_phases`.  It looks like this is happening because the examples are also in the docstring. I recommend deleting them there, but check with Rob first.

(Alternatively, I don't know if there is a way to modify the eval-rst method so that it doesn't pull in any extraneous stuff from the docstring, but that is an alternative.

Reply:
Tagging @rfalck to get his take on this comment

Reply:
remove from the docstring

Reply:
Removed